content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -1,97 +1,99 @@
-
-### 3. Enable Kubeflow
+    **Optional:** To enable NVIDIA GPU hardware support, also run `microk8s.enable gpu`.
 
-- Deploy Kubeflow with the command:
+4. Deploy Kubeflow by running this command:
 
     ```
     microk8s.enable kubeflow
     ```
 
-- The deployment process may take a few minutes. Once completed, the script will print out the port number and credentials to access the Kubeflow dashboard.
+    The deployment process may take a few minutes. Once it is complete, the script will print out the port number and credentials to access the Kubeflow dashboard.
 
-## Access Kubeflow dashboard
+
+## Accessing the Kubeflow dashboard
 
 ### On your Linux machine
-If you installed Microk8s directly on your Linux machine, (1) open a web browser window and (2) access the link provided after you enable Kubeflow, e.g. `10.64.140.43.xip.io` (see previous step).
 
-### On Multipass or a virtual machine
-When running Microk8s on Multipass or a virtual machine, create a SOCKS proxy to access the Kubeflow dashboard, as follows:
+If you installed MicroK8s directly on your Linux machine, you can view the Kubeflow dashboard as follows:
+
+1. Open a web browser window.
+2. Access the link provided after you have enabled Kubeflow (for example,
+   `10.64.140.43.xip.io`).
+
+### On Windows, macOS, Multipass or a virtual machine
 
-* Logout from the current session using the `exit` command.
-* Re-establish connection to the machine using `SSH`, enabling SOCKS proxy with the `-D9999` parameter. Examples:
+When running MicroK8s on Windows, macOS, Multipass or a virtual machine, you need to create a SOCKS proxy to access the Kubeflow dashboard:
+
+1. Logout from the current session in your terminal using the `exit` command.
+
+2. Re-establish connection to the machine using `SSH`, enabling SOCKS proxy with the `-D9999` parameter.
+
+    In the VM case, run the following command, where `<machine_public_ip>` is your machine's public IP:
 
     ```
     ssh -D9999 ubuntu@<machine_public_ip>
     ```
 
-    or find multipass IP with `multipass list` and connect with:
+    On Windows, macOS or Multipass, you can check for the IP first with:
+
+    ```
+    multipass list`
+    ```
 
+    and then, run this command, replacing `<multipass_public_ip>` with that IP:
     ```
     ssh -D9999 multipass@<multipass_public_ip>
     ```
 
-* On your computer, go to `Settings > Network > Network Proxy`, and enable SOCKS proxy pointing to: `127.0.0.1:9999`.
+3. In your Linux operating system, go to **Settings** > **Network** > **Network Proxy**, and enable SOCKS proxy pointing to: `127.0.0.1:9999`.
 
-* Finally, (1) open a new web browser window and (2) access the link provided after you enable Kubeflow, e.g. `10.64.140.43.xip.io` (see previous step).
+4. Finally, access the Kubeflow dashboard by:
+    1. Opening a new web browser window.
+    2. Access the link provided after you have enabled Kubeflow (for example, `10.64.140.43.xip.io`).
```

Comment:
Maybe this should be "Accessing". Other than that, lgtm

Reply:
Nice catch! Solved! 🚀